src/content/docs/pl/getting-started.mdx

```diff
@@ -2,108 +2,63 @@
-
-
-## Rozbuduj swój projekt Astro
-
-🧰 Zacznij swój następny projekt od [gotowego motywu](https://astro.build/themes/)
-
-🧰 Dostosuj swoją stronę za pomocą oficjalnych i społecznościowych [wtyczek i komponentów](https://astro.build/integrations/).
-
-🧰 Zainspiruj się, odwiedzając naszą [galerię stron Astro](https://astro.build/showcase/).
-
-*... zobacz nasz [przewodnik po korzystaniu z integracji](/pl/guides/integrations-guide/)*.
-
-
-
-## Dołącz do naszej społeczności
-
-Dołącz do nas na [Discordzie Astro](https://astro.build/chat), aby podzielić się z i uzyskać pomoc od aktywnej, przyjaznej społeczności!
-
-💬 Przywitaj się na naszym kanale `#general`!
-
-💬 Zadaj pytanie naszemu Zespołowi Wsparcia na kanale `#support`!
-
-💬 Podziel się tym, nad czym pracujesz na kanale `#showcase`!
-
-
-## Co nowego?
-
-[Blog Astro](https://astro.build/blog/)
-
-[Lista zmian w Astro](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
-
-## Wesprzyj nas
-
-Ta dokumentacja jest dostarczana do ciebie przez wszystkich tych pomocnych ludzi. [Dołącz do nas na GitHubie!](https://github.com/withastro/docs)
-
-<ContributorList githubRepo="withastro/docs" />
+import { CardGrid } from '@astrojs/starlight/components'
+import Card from '~/components/Landing/Card.astro'
+import ListCard from '~/components/Landing/ListCard.astro'
+import SplitCard from '~/components/Landing/SplitCard.astro'
+import Discord from '~/components/Landing/Discord.astro'
+
+<CardGrid>
+  <Card title="Co stworzysz z Astro?" icon="laptop">
+    Eksploruj [startowe arkusze Astro](https://astro.build/themes/) aby znaleźć blogi, portfolia, dokumentacje, strony startowe, SaaS, marketing, strony ecommerce oraz więcej!
+  </Card>
+
+  <Card title="Wycieczka z przewodnikiem" icon="star">
+    Ukończ nasz startowy [kurs gdzie zbudujesz bloga](/pl/tutorial/0-introduction/), aby nauczyć się podstaw oraz stworzyć swoją pierwszą stronę z Astro.
+  </Card>
+
+  <SplitCard title="Zacznij nowy projekt" icon="rocket">
+    ```sh
+    # stwórz nowy projekt z npm
+    npm create astro@latest
+    ```
+    
+    Nasz [przewodnik instalacji](/pl/install/auto/) zawiera instrukcje mówiące krok po kroku jak zainstalować Astro za pomocą instalatora CLI, jak stworzyć nowy projekt z istniejącego repozytorium Astro na GitHubie, oraz jak ręcznie zainstalować Astro.
+  </SplitCard>
+
+  <ListCard title="Nauka" icon="open-book">
+    - [Główne funkcje Astro](/pl/concepts/why-astro/)
+    - [Architektura wysp](/pl/concepts/islands/)
+    - [Komponenty Astro](/pl/basics/astro-components/)
+    - [Składnia plików Astro](/pl/basics/astro-syntax/)
+  </ListCard>
+
+  <ListCard title="Rozszerzanie" icon="puzzle">
+    - [Dodaj integracje jak React czy Tailwind](/pl/guides/integrations-guide/)
+    - [Twórz otypowane kolekcje zawartości](/pl/guides/content-collections/)
+    - [Ulepsz nawigację dzięki View Teansitions](/pl/guides/view-transitions/)
+    - [Połącz bezgłowy CMS ze swoim projektem](/pl/guides/cms/)
```

Comment:
I suggest using original form - headless  CMS. It sounds super weird in polish 😅

Reply:
Hmmm when I really think about it, yeah it kinda does